### PR TITLE
Adds FreshRSS service

### PR DESCRIPTION
**Pull request by kinduff**
Adds support for [FreshRSS](https://freshrss.org). I've been running this for a couple weeks and works like a charm.

### CODE REVIEW

**Review by QEDeD — APPROVED** [review]
LGTM - for what that's worth

File: docs/services/freshrss.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# FreshRSS
+
+[FreshRSS](https://freshrss.org) is a self-hosted RSS and Atom feed aggregator. It is lightweight, easy to work with, powerful, and customizable.
+
+## Dependencies
+
+This service requires the following other services:
+
+-   a [Traefik](traefik.md) reverse-proxy server
+-   an optional [Postgres](postgres.md) database, but [SQLite](https://www.sqlite.org/) is set by default.
```
Comment (spantaleev) [inline docs/services/freshrss.md:10]:
```suggestion
-   an optional [Postgres](postgres.md) database, but FreshRSS will default to [SQLite](https://www.sqlite.org/) if you don't have Postgres enabled
```

File: group_vars/mash_servers [hunk 2]
```diff
@@ -1080,6 +1088,51 @@ focalboard_container_labels_traefik_tls_certResolver: "{{ devture_traefik_certRe
...
+########################################################################
+
+freshrss_enabled: false
+
+freshrss_identifier: "{{ mash_playbook_service_identifier_prefix }}freshrss"
+
+freshrss_base_path: "{{ mash_playbook_base_path }}/{{ mash_playbook_service_base_directory_name_prefix }}freshrss"
+
+freshrss_uid: "0"
+freshrss_gid: "33"
```
Comment (spantaleev) [inline group_vars/mash_servers:1104]:
```suggestion
# freshrss_uid and freshrss_gid are intentionally not being set here.
# FreshRSS can only work with a specific user and group, as hardcoded in the role defaults.
```

**Comment by kinduff** [comment]
@spantaleev Done! Thanks!

**Pull request merged**